server/server.py

```
@@ -0,0 +1,240 @@
+                challenger = next(
+                    (ch for ch, (tgt, _) in _pending_challenges.items()
+                     if tgt == name),
+                    None,
+                )
+                if not challenger or challenger not in _players:
+                    continue
+
+                _, settings = _pending_challenges.pop(challenger)
+
+                if accepted:
+                    seed = random.randint(0, 2 ** 31 - 1)
```

Ruff (0.15.7)

[error] Standard pseudo-random generators are not suitable for cryptographic purposes

(S311)